.github/workflows/alerting-release-e2e-workflow.yml

```diff
@@ -0,0 +1,17 @@
+name: Alerting Release tests workflow in Bundled OpenSearch Dashboards
+on:
+  pull_request:
+    branches:
+      - main
+      - dev-*
+  push:
```

Comment:
Do we on need on push?

Reply:
seems no too much value for running on push. Let me remove these.